lib/card.rb

```diff
@@ -38,4 +38,25 @@ class Card < ActiveRecord::Base
   TRACKED_FIELDS = %w(name type_id db_content trash)
   Loader.load_mods if count > 0
 
+
+  def self.find_each(options = {})
+    if block_given?
+      find_in_batches(options) do |records|
+        records.each { |record| yield record }
+      end
+    else
+      super(options)
+    end
+  end
```

Comment:
Weird. I wouldn't think that would be true, and last night I could swear I
tried it without it and it worked.

I'll check again.

On Sat, Jan 10, 2015 at 10:00 AM, xithan <notifications@github.com> wrote:

> In lib/card.rb
> <https://github.com/wagn/wagn/pull/280#discussion-diff-22760249>:
>
> > @@ -38,4 +38,25 @@ class Card < ActiveRecord::Base
> >    TRACKED_FIELDS = %w(name type_id db_content trash)
> >    Loader.load_mods if count > 0
> >
> > +
> > +  def self.find_each(options = {})
> > +    if block_given?
> > +      find_in_batches(options) do |records|
> > +        records.each { |record| yield record }
> > +      end
> > +    else
> > +      super(options)
> > +    end
> > +  end
>
> It doesn't change anything but if I don't overwrite find_each then it
> calls the original find_in_batches instead of Card.find_in_batches
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wagn/wagn/pull/280/files#r22760249>.
>



-- 
Ethan McCutchen
One of the Wagneers, Wagn.org

Wagn. How pioneers roll.

s: ethan.mccutchen
t: @intogreater